sd-validation/test/json-validation-test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env mocha */
+
+const assert = require('assert');
+const validateJSONLD = require('../');
+
+describe('JSON validation', () => {
+  it('reports missing closing bracket', async () => {
+    const errors = await validateJSONLD(`{
+      "test": "test"
+    `);
+
+    assert.equal(errors.length, 1);
+    assert.equal(errors[0].path, 2);
+    assert.ok(errors[0].message.indexOf('Expecting \'}\'') === 0);
+  });
+
+  it('reports missing coma', async () => {
+    const errors = await validateJSONLD(`{
+      "test": "test"
+      "test2": "test2"
+    }`);
+
+    assert.equal(errors.length, 1);
+    assert.equal(errors[0].path, 2);
+    assert.ok(errors[0].message.indexOf('Expecting \'EOF\', \'}\', \':\', \',\', \']\'') === 0);
```

Comment:
i don't care a lot but backtick string would make this easier to write and read. :)

Reply:
Fixed by Konrad.